File: app/src/apiProd/java/cz/vutbr/fit/brnogo/injection/UrlModule.java
package cz.vutbr.fit.brnogo.injection;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;
import okhttp3.HttpUrl;

@Module
public class UrlModule {

	@Provides
	@Singleton
	public HttpUrl url() {
		return HttpUrl.parse("http://localhost:8080/api/");
	}
}

File: app/src/apiProd/java/cz/vutbr/fit/brnogo/injection/UrlModule.kt
package cz.vutbr.fit.brnogo.injection

import dagger.Module
import dagger.Provides
import okhttp3.HttpUrl
import javax.inject.Singleton

@Module
class UrlModule {

	@Provides
	@Singleton
	fun url(): HttpUrl {
		return HttpUrl.parse("https://brnogoserver.localtunnel.me/brnogo/api/")!!
	}
}
